src/actions/projects.js

```diff
@@ -121,16 +123,15 @@ const getAvatarsForProjects = projects => {
     })
 }
 
-const getMuseumRoleForProjects = projects => {
-    return projects.map(project => {
-        return apiClient.type('project_roles')
-            .get({id: project.links.project_roles})
-            .then((roles) => {
-                roles.forEach(role => {
-                    project.in_museum_mode = project.in_museum_mode || role.roles.includes('museum')
-                })
-            })
-    })
+export const tagMuseumRoleForProjects = projects => {
+    return apiClient.type('projects')
+      .get({ current_user_roles: 'museum' })
```

Comment:
This looks great! I wish I had known that `current_user_roles` was an option when I was writing this the first time. Is there some documentation or another way that I can figure out things like this in the future?

Reply:
It should probably be in the Apiary, but doesn't appear to be there. We're also using `current_user_roles` at the top of this script, but I'm not sure where/if documentation on this lives anywhere.

Reply:
Unfortunately it's an undocumented Panoptes feature. It works on any controlled resource, so projects, organizations, and collections for an authenticated requests. The filter should include an array of the user roles you want to filter against. Here's its code if that is at all helpful: https://github.com/zooniverse/Panoptes/blob/4cfe0e6e8bf7b3edd98490416caedb116207753a/app/controllers/concerns/filter_by_current_user_roles.rb

And an example of its use on PFE:
https://github.com/zooniverse/Panoptes-Front-End/blob/21cf42485929a62938112d5e2d4bca1a6702b00e/app/pages/home-for-user/recent-collections.jsx#L48

Reply:
Thank you @srallen !

Reply:
> Unfortunately it's an undocumented Panoptes feature.

This is unfortunate, looks like we failed to add the docs after this came in, https://github.com/zooniverse/Panoptes/issues/864

FWIW - apiary  got bought by oracle a while back, i'm not sure we've still got access to update it either :( - though feel free to try and update https://github.com/zooniverse/Panoptes/blob/7d8a941fd7bc190e8dc4646da20b11d8b0a251a1/apiary.apib#L1122 to add the filter examples in. If that merges in theory the docs will update on Apiary. 

FYI - The plan is to shift to our own documentation scheme in https://github.com/zooniverse/Panoptes/pull/3050 but we haven't had any traction on porting those docs.

> It works on any controlled resource, so projects, organizations, and collections for an authenticated requests.

And to clarify - it only works on `projects`, `organizations`, and `collections`.